README.md

```diff
@@ -1 +1,52 @@
-# token-seeder
\ No newline at end of file
+# Token Seeder
+
+| Key      | Value                                                                                    |
+|----------|------------------------------------------------------------------------------------------|
+| Version  | 1.0.0                                                                                    |
+| Source   | https://github.com/VNG-Realisatie/token-seeder                                           |
+| Keywords | ZGW, tooling                                                                             |
+| Related | https://github.com/VNG-Realisatie/token-issuer, https://gitlab.com/commonground/ri/infra/ |
+
+
+## Introductie
+
+Binnen het Nederlandse gemeentelandschap wordt zaakgericht werken nagestreefd.
+Om dit mogelijk te maken is er gegevensuitwisseling nodig. De kerngegevens van
+zaken moeten ergens geregistreerd worden en opvraagbaar zijn. Om dit proces te ondersteunen is er een
+authenticatie module waarbij tokens gegeneerd worden. Om dit proces ook in een kubernetes cluster te ondersteunen
+zonder admin is er gekozen voor een seeder die de juiste data aanmaakt.
```

Comment:
```suggestion
Binnen het Nederlandse gemeentelandschap wordt zaakgericht werken nagestreefd.
Om dit mogelijk te maken is er gegevensuitwisseling nodig. De kerngegevens van
zaken moeten ergens geregistreerd worden en opvraagbaar zijn. Om dit proces te ondersteunen is er een
authenticatie module waarbij tokens gegeneerd worden. Om dit proces ook in een kubernetes cluster te ondersteunen zonder admin is er gekozen voor een seeder die de juiste data aanmaakt.
```
Om het zaakgericht werken in de Common Ground architectuuur te ondersteunen zijn de [API's voor Zaakgericht werken](https://github.com/VNG-Realisatie/gemma-zaken) ontwikkeld. Onderdeel van deze API's is een autorisatie module, de [Autorisaties API](https://github.com/VNG-Realisatie/autorisaties-api) welke tokens gebruikt. Deze tokens dienen genereerd en ingericht te worden in de verschillende API's. Om dit proces ook in een kubernetes cluster te ondersteunen zonder admin is er gekozen voor een seeder die de juiste data aanmaakt.

Reply:
Tekst suggestie  :-)